code/modules/projectiles/guns/energy/special.dm

```diff
@@ -556,3 +556,27 @@
 	fire_delay = 20
 	accuracy = 40
 	muzzle_flash = 10
+
+//Pocket Protector
+/obj/item/weapon/gun/energy/tiny_gun
+	name = "PocketProtector228"

```

Comment:
Does not follow the convention in naming guns, change the name to something more generic, please.